dns/bind/Makefile

```
@@ -1,6 +1,5 @@
 PLUGIN_NAME=		bind
-PLUGIN_VERSION=		1.4
-PLUGIN_REVISION=        1
+PLUGIN_VERSION=		1.5
```

```suggestion
PLUGIN_VERSION=		1.5.d
```

Let's reserve "1.5" for the release :)